core/utils/poly-look/src/react/chips/chips.jsx

```diff
@@ -1,21 +1,131 @@
-import React from "react";
+import React, { useState } from "react";
 
 import "./chips.css";
 
-const Chips = ({ chipsContent, activeChips, onChipClick, theme }) => {
+const chipsId = (e) => e.id || e.title || e;
+
+const othersId = "others";
+const allId = "all";
+
+/**
+ *
+ * Chips group
+ * @param {string[] || Object[]} chipsContent -  Content of the Chips
+ * @param {Object} [chipsContent.id] - Id of the chip (if chipsContent is an array of strings they act as id)
+ * @param {Object} [chipsContent.title] - Alternative for id of the chip (if chipsContent is an array of strings they act as id)
+ * @param {Object} [chipsContent.translation] - Translation of the chip (if chipsContent is an array of strings they act as translation)
+ * @param {string[]} defaultActiveChips - Chips that are active on load
+ * @param {callback} onChipClick - Chips onClick function (id of clicked chip, all active chips ids)
+ * @param {boolean = true} [exclusive] - Determines whether chips are active exclusively
+ * @param {string} [theme] - Sets the theme in this component (preferably done in parent component)
+ * @param {Object} [all] - Indicated whether an all chip exists (exclusive to the other chips, activates all)
+ * @param {string = "All"} [all.translation] - Translation for chips text ("All")
+ * @param {Object} [others] - Indicated whether an others chip exists (groups multiple chips in "Others" chip)
+ * @param {string} [others.translation] - Translation for chips text ("Others")
+ * @param {string[]} [others.ids] - Ids of the chips grouped by others
+ * @param {boolean} [others.exclusive] - Indicates whether the others chips is exclusive to the rest
+ * @returns
+ */
+const Chips = ({
+  chipsContent,
+  defaultActiveChips,
+  onChipClick,
+  exclusive = true,
+  theme,
+  all,
+  others,
+}) => {
+  const chipsContentIds = chipsContent.map((e) => chipsId(e));
+
+  let extendedChipsContent = [...chipsContent].filter((e) =>
+    others ? others.ids.indexOf(chipsId(e)) == -1 : e
+  );
+  if (others) {
+    extendedChipsContent.push({
+      id: othersId,
+      translation: others.translation || "Others",
```

Comment:
This kind of calls for exclusive logic for these kind of chips, I think.
I can't really figure out how, I'd have to look at the whole implementation.